File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/EdneListarUsuarioXDocumentoRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class EdneListarUsuarioXDocumentoRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private List<EdneListarUsuarioXDocumentoDetRequest> listaUsuarioDocumento;
	private String campNroDoc;
	private String campTipDoc;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/edne-listar-usuario-x-documento.request.ts
import { EdneListarUsuarioXDocumentoDetRequest } from "./edne-listar-usuario-x-documento-det.request";

export class EdneListarUsuarioXDocumentoRequest {
    listaUsuarioDocumento: EdneListarUsuarioXDocumentoDetRequest[];
    campNroDoc: string;
    campTipDoc: string;
}